src/security/ldap/pom.xml

```diff
@@ -40,6 +40,16 @@
       <groupId>org.springframework.ldap</groupId>
       <artifactId>spring-ldap-test</artifactId>
       <scope>test</scope>
+      <exclusions>
```

Comment:
Why do we need this?

Reply:
The "ldap-tests" module includes a very old version of Spring-Batch which is incompatible with the version used by the Backup-&-Restore module. You might end up with a method clashing and odd errors like "method not found" on Classes.  In some circumstances, it also prevents GeoServer to start.

Reply:
Good, merging then. I see you already backported to 2.13.x, can you also backport for 2.14.x?

Reply:
Sure